components/form/builder/src/Select/Autosuggest/index.js

```diff
@@ -24,13 +24,18 @@ const AutosuggestSelect = ({select, tabIndex, onChange, size, errors}) => {
   const {datalist = []} = select
   const fromTextToValueWithDatalist = fromTextToValue(datalist)
   const fromValueToTextWithDatalist = fromValueToText(datalist)
-  const text = fromValueToTextWithDatalist(select.value)
-  const [stateText, setStateText] = useState(text)
+  const textFromValueProp = fromValueToTextWithDatalist(select.value)
+  const [localStateText, setLocalStateText] = useState('')
+
+  // case: a value is forced from outside the component
+  // if text from prop is different from current local state text, update local text state
+  if (textFromValueProp && textFromValueProp !== localStateText)
```

Comment:
This is going to cause a re-render of the component the first time if the `textFromValueProp` is different to `''`.

Reply:
it is exactly what I want. Because If I want to set a text value from outside the component (textFromValueProp) I will need a re-render in order to print this textFromValueProp in the component

Reply:
I know this sync is kind of weird, but form builder works with the numerical ids and the autosuggest works with the text that is in the input...

Reply:
I was only noticing it :). Normally this is a smell that something is not being handled correctly in React... I dunno if maybe you could try to approach the problem by using a `useEffect` to be fired only when the dependency is changed and avoid executing it in the first render 🤔 .